src/sentry/static/sentry/app/views/organizationDiscover/result/pagination.jsx

```diff
@@ -9,34 +9,72 @@ export default class Pagination extends React.Component {
   static propTypes = {
     getNextPage: PropTypes.func.isRequired,
     getPreviousPage: PropTypes.func.isRequired,
-    previous: PropTypes.string,
-    next: PropTypes.string,
+    baseQuery: PropTypes.shape({
+      query: PropTypes.shape({
+        limit: PropTypes.number.isRequired,
```

Comment:
Probably not required if we can use data length to determine the result range

Reply:
yeah, i only have it for whether or not the 'next' button should be disabled